korman/exporter/mesh.py

```diff
@@ -267,22 +288,17 @@ def _export_geometry(self, bo, mesh, materials, geospans):
                         geoVertex.normal = hsVector3(*tessface.normal)
 
                     geoVertex.color = hsColor32(*vertex_color)
-                    geoVertex.uvs = [hsVector3(uv[0], 1.0 - uv[1], 0.0) for uv in uvws]
+                    uvs = list([hsVector3(uv[0], 1.0 - uv[1], 0.0) for uv in uvws])
+                    if bumpmap is not None:
+                        uvs.append(dPosDu)
+                        uvs.append(dPosDv)
```

Comment:
oh! thanks, that makes sense now. I still don't like the idea of having to convert the UVs from a tuple to a list over and over and over again, but so be it, I guess...